packages/esm-patient-chart-app/src/visit/hooks/useUpcomingAppointments.tsx

```diff
@@ -17,13 +18,14 @@ export interface AppointmentPayload {
   dateHonored?: string;
 }
 
-export function saveAppointment(appointment: AppointmentPayload, abortController: AbortController) {
-  return openmrsFetch(`${restBaseUrl}/appointment`, {
+export const updateAppointmentStatus = async (toStatus: string, appointmentUuid: string) => {
+  const abortController = new AbortController();
```

Comment:
Previously this took in an abortController, now you are instantiating one.  I think there is a `useAbortController` hook in esm-framework.  Perhaps you are meant to use that? @ibacher / @denniskigen ?